client/src/pages/Navbar.js

```diff
@@ -35,45 +49,62 @@ export default function NavigationBar() {
     setAnchorEl(event.currentTarget);
   };
 
+  const handleLogout = e => {
+    e.preventDefault();
+    localStorage.clear();
+    window.location.href = "/";
+  }
+
+  const token = localStorage.getItem('jwtToken');
+  let buttons;
+
+  if (token) {
+    buttons = <div>
+                <Button component={Link} to={"/profile"}>BECOME A SITTER</Button>
+                <Button>My Sitters</Button>
+                <Button>Messages</Button>
+                <IconButton aria-label="avatar" onClick={handleClick}>
+                  <Avatar alt="Remy Sharp" src={require("../images/07cc6abd390ab904abbf31db5e6ea20357f8b127.png")} className={classes.bigAvatar}/>
+                </IconButton>
+                <Menu
+                      id="menu-appbar"
+                      anchorEl={anchorEl}
+                      anchorOrigin={{
+                        vertical: 'bottom',
+                        horizontal: 'right',
+                      }}
+                      keepMounted
+                      transformOrigin={{
+                        vertical: 'top',
+                        horizontal: 'right',
+                      }}
+                      open={open}
+                      onClose={handleClose}
+                    >
+                      <MenuItem><a href="./profile">Profile</a></MenuItem>
+                      <MenuItem onClick={handleClose}>My account</MenuItem>
+                      <MenuItem onClick={handleLogout}>Log Out</MenuItem>
+                </Menu>
+              </div> 
+  } else {
+    buttons = <div>
+                <Button className={classes.menuButton} color="inherit" component={Link} to={"/profile"}>BECOME A SITTER</Button>
+                <Button className={classes.menuButton} variant="outlined" color="secondary" component={Link} to={"/login"}>Log In</Button>
+                <Button variant="contained" color="secondary" component={Link} to={"/signup"}>Sign Up</Button>
+              </div>
+  }
+
   return (
     <div className={classes.root}>
-      <AppBar position="static" color="inherit">
+      <AppBar className={ token ? classes.loggedInNavbar : classes.logInNavbar}>
+      {/* <AppBar position="fixed" className="transparentNavbar"> */}
```

Comment:
You may want to take this out before merging

Reply:
yes, I removed the comment out